mothership/src/org/labkey/mothership/MothershipManager.java

```diff
@@ -426,6 +426,7 @@ else if (StringUtils.isEmpty(currentValue))
         ObjectMapper mapper = new ObjectMapper();
         try
         {
+            log.info("Merging JSON. Old is " + currentValue + " characters, new is " + newValue.length());

```

Comment:
Should this be logged at debug level or are you just planning on removing it before merging?
```suggestion
            log.debug("Merging JSON. Old is " + currentValue + " characters, new is " + newValue.length());

```

Reply:
Also, did you mean `currentValue.length()`?

Reply:
Yes